src/components/Card/Card.tsx

```diff
@@ -1,24 +1,35 @@
 import './Card.scss'
 
-interface IProps {
+interface IPropsData {
     title: string;
     description: string;
     id: string;
 }
 
-const Card = (props: { data: IProps[]; }) => {
-const {data} = props
+interface IProps {
+    data: IPropsData[]
+    removeCardHandler: (id: string) => void;
+    editCardHandler: (id: string) => void;
+    isEdit:boolean
+}
+
+const Card = (props: IProps) => {
+const {data,removeCardHandler, editCardHandler,isEdit } = props
 
     return (
         <div className="cards-container">
-        {data.map((el: IProps) => (
+        {data.map((el: IPropsData) => (
             <div className="card" key={el.id}>
+            {isEdit ? <span id={el.id} onClick = {()=>removeCardHandler(el.id) }>&#10060;</span> : null}
```

Comment:
This should be a global btn, like turn off 'edit mode'. Not for each card.

Reply:
done